src/System.Net.HttpListener/tests/HttpListenerAuthenticationTests.cs

```diff
@@ -22,15 +22,16 @@ public class AuthenticationTests : IDisposable
         private HttpListenerFactory _factory;
         private HttpListener _listener;
 
-        public AuthenticationTests()
+        public HttpListenerAuthenticationTests()
         {
             _factory = new HttpListenerFactory();
             _listener = _factory.GetListener();
         }
 
         public void Dispose() => _factory.Dispose();
 
-        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // Managed implementation connects successfully.
+        // [ActiveIssue(20840, TestPlatforms.Unix)] // Managed implementation connects successfully.
```

Comment:
Do we need this comment? 

Issue #20840 is open, if managed implementation connects successfully, I think that issue can be closed.